### ISSUE TITLE
Can't resolve 'frappe-gantt' (Angular app)

### ISSUE THREAD

**Issue by frankdenouter** [frappe/gantt#429]
Thank you for all the work! I saw you fixed a lot bugs 2 days ago.

But when I upgraded from commit #b7deffd53954ebae5c6dd77bdab54551ca1d17e8  to tag 0.8.0 I could not compile anymore. I got the error: `Error: Module not found: Error: Can't resolve 'frappe-gantt' in 'C:\Users\fdo\projects\myfancyapp\src\app\components\view`

I had it all working on  commit #b7deffd53954ebae5c6dd77bdab54551ca1d17e8 (`npm i git+https://github.com/frappe/gantt#b7deffd53954ebae5c6dd77bdab54551ca1d17e8`) with @types/frappe-gant and  with a reference to the correct .css in de angular.json file and the `"allowSyntheticDefaultImports": true` in tsconfig.json.

Can you provide a solution?

**Comment by frankdenouter** [comment]
I created a sample project: https://stackblitz.com/edit/angular-vmh5gi 

I do not get the error here, but the styling is gone.

**Comment by safwansamsudeen** [comment]
Hey, is there any reason why you closed this?

That's weird - could we get on a call, if you want, to figure it out?

My TZ is IST.

**Comment by frankdenouter** [comment]
Sorry I had not left a comment when closing this issue. I don't need any more assistance. I was looking for a solution to house the library in an Angular application, but ran into many things.
In my search, I found ‘vis-timeline’, which better suited my needs and has better integration with Angular.

On the internet, I found few examples with Angular and frappe-gantt, so I don't think there is much demand for it, so I thought it would be a waste of your time to leave the issue open.

Still, I want to thank you for the effort of replying to this issue and wish you good luck with frappe-gantt.
